service-base/src/test/java/org/eclipse/hono/service/tenant/BaseTenantServiceTest.java

```diff
@@ -147,4 +147,5 @@ public void setConfig(final ServiceConfigProperties configuration) {
             }
         };
     }
+
```

Comment:
?

Reply:
I assume you refer to the empty line, I removed it.